packages/wonder-blocks-grid/components/cell.js

```diff
@@ -15,21 +15,31 @@ type Props = {
     /** The number of columns this cell should span on a Large Grid. */
     largeCols?: number,
     /** The number of columns this should should span by default. */
-    cols?: number | ((size: GridSize) => number),
+    cols?: number | ((size: MediaSize) => number),
     /**
      * The child components to populate inside the cell. Can also accept a
-     * function which receives the `gridSize`, `totalColumns`, and cell
+     * function which receives the `mediaSize`, `totalColumns`, and cell
      * `width` and should return some React Nodes to render.
      */
     children:
         | React.Node
         | (({
-              gridSize: GridSize,
+              mediaSize: MediaSize,
               totalColumns: number,
               cols: number,
           }) => React.Node),
     /** The styling to apply to the cell. */
     style?: any,
+    /**
+     * The size of the media layout being used. Populated by MediaLayoutWrapper.
+     * @ignore
+     */
+    mediaSize: MediaSize,
+    /**
+     * The current media layout spec being used. Populated by MediaLayoutWrapper.
+     * @ignore
```

Comment:
Does this hide them in the generated docs?

Reply:
Yes! I find this to be convenient as it still explains what it is if you're reading the code but we don't need to describe it in the official docs as no one is really expected to use it.